static/bidder-info/admatic.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "http://{{.Host}}"

```

Comment:
Adapters are not allowed to use an endpoint passed through from the request.

Please see the [Module Rules](https://docs.prebid.org/dev-docs/module-rules.html).  This is listed as Rule 7 in the "Strictly Enforced Rules" section under "Global Module Rules".